browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2334,17 +2332,16 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			controls['button'] = button;
 			if (builder.options.noLabelsForUnoButtons !== true) {
-				var label = L.DomUtil.create('label', 'ui-content unolabel', button);
+				label = L.DomUtil.create('label', 'ui-content unolabel');
 				label.htmlFor = buttonId;
 				label.textContent = builder._cleanText(data.text);
 				builder._stressAccessKey(label, button.accessKey);
 				controls['label'] = label;
-				$(div).addClass('has-label');
+				$(button).addClass('has-label');
```

Comment:
can we not use jQuery as we rework it anyway?

Reply:
Yes, I do not like to use JQuery, so I followed the pattern since it is used all around us =)